### ISSUE TITLE
Uncaught RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by yragunathan** [ckeditor/ckeditor-dev#2007]
## Type of report

Uncaught RangeError: Maximum call stack size exceeded error while using the rich text editor in Drupal 7 after the update of the new CKeditor 4.9.2

Uncaught RangeError: Maximum call stack size exceeded
    at String.replace (<anonymous>)
    at prepareDataForWysiwygMode (plugin.js?t=I3I8:89)
    at e.upcast (plugin.js?t=I3I8:115)
    at Array.<anonymous> (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:414)
    at iterator (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:1632)
    at CKEDITOR.htmlParser.element.forEach (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:687)
    at CKEDITOR.htmlParser.element.forEach (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:687)
    at CKEDITOR.htmlParser.element.forEach (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:687)
    at CKEDITOR.htmlParser.element.forEach (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:687)
    at CKEDITOR.htmlParser.element.forEach (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:687)

Another issue:
When changing from full html to filtered html or vice versa the maximum call stack size exceeded. Also another error is shown as below:
js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:1643 Uncaught TypeError: Cannot read property 'editables' of undefined
    at js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:1643
    at CKEDITOR.htmlParser.element.forEach (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:687)
    at CKEDITOR.htmlParser.element.forEach (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:687)
    at CKEDITOR.htmlParser.element.forEach (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:687)
    at CKEDITOR.htmlParser.element.forEach (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:687)
    at a.<anonymous> (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:1642)
    at a.m (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:394)
    at a.<anonymous> (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:396)
    at a.CKEDITOR.editor.CKEDITOR.editor.fire (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:397)
    at CKEDITOR.htmlDataProcessor.toDataFormat (js_52fl9s6SwjJEPHP3e_OviG1hGzgTWd1LHC3wPQzgboE.js:709)

##Expected result:
Only after changing from CKEditor 4.5.4 to 4.9.2 the editor throws JS error. Need suggestion to fix it ASAP.

**Comment by jacekbogdanski** [comment]
Hi, It's been a while since we last heard from you. We're closing this issue for now. Still, feel free provide us requested feedback, so that we can reopen this issue.
